math/OpenBLAS/Portfile

```diff
@@ -27,6 +23,36 @@ platform darwin i386 {
     compiler.fallback-append macports-clang-5.0 macports-clang-6.0 macports-clang-7.0
 }
 
+#Define target based on version of OS
+#OS 10.5 supports down to Intel Core Solo (Intel) or PowerPC G4
+if {${os.major} >= 9} { 
```

Comment:
Should this be <= 9 ? I don't see a target here for OSX 10.4 PPC ... maybe I'm just missing something?

Reply:
That's on purpose. 10.4 and below also support PowerPC G3, for which OpenBLAS does not have any target, so I fall back to use the native variant as default. 

Another point, although less important, is that we don't have bots for versions lower than 10.5, so binaries would not be provided automatically anyway.

Reply:
OK; thx. So do you want to make this one `== 9` instead? I think then all of the OSX versions for pre-built-binaries would be covered.

Reply:
Done